pnpm-lock.yaml

```diff
@@ -5195,13 +5195,25 @@ packages:
     engines: {node: '>=16'}
     dev: true
 
+  /@edge-runtime/primitives@3.1.1:
+    resolution: {integrity: sha512-ROO22py+KdAfzqWZu6CtVMC4qV6mS0W1jPI51jGXE+uenyBUN7cQTWB9ReQc8Bm4cnjqmhajvpqEx3j7Y9iSOg==}
+    engines: {node: '>=16'}
+    dev: true
+
   /@edge-runtime/vm@3.1.0:
```

Comment:
Is it expect that both `vm@3.1.0` and `vm@3.1.1` are used?

Reply:
it should be the same 🤔